tests/acceptance/features/apiActivities/shareActivities.feature

```diff
@@ -497,3 +497,173 @@ Feature: check share activity
         }
       }
       """
+
+
+  Scenario: sharer checks (add/remove) member activity
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has sent the following space share invitation:
+      | space           | new-space    |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Space Editor |
+    And user "Alice" has removed the access of user "Brian" from space "new-space"
+    When user "Alice" lists the activities of space "new-space" using the Graph API
```

Comment:
we can have two separate scenarios for addition and removal
```suggestion
      | permissionsRole | Space Editor |
    When user "Alice" lists the activities of space "new-space" using the Graph API
```

Reply:
Sure, I will separate them into two scenarios.